### PR TITLE
Fix various makefile issues with JDK11

### PR DESCRIPTION
**Pull request by ChengJin01**
The changes are to resolve all makefile issues
when compiling JDK11 on Windows.

Signed-off-by: CHENGJin <jincheng@ca.ibm.com>

### CODE REVIEW

File: closed/make/copy/Copy-java.base.gmk [hunk 1]
```diff
@@ -29,6 +29,7 @@ ALT_JVMCFG_SRC := $(TOPDIR)/closed/src/java.base/conf/jvm.cfg
 override-jvm.cfg : $(LIB_DST_DIR)/jvm.cfg
 	$(ECHO) "Overriding jvm.cfg with J9VM version"
 	$(CP) $(ALT_JVMCFG_SRC) $(LIB_DST_DIR)/jvm.cfg
+	$(CP) $(ALT_JVMCFG_SRC) $(OUTPUTDIR)/jdk/lib/jvm.cfg
```
Comment (pshipton) [inline closed/make/copy/Copy-java.base.gmk]: Why is this required?

File: closed/OpenJ9.gmk [hunk 2]
```diff
@@ -378,6 +378,12 @@ endif
 	@$(MKDIR) -p $(MODULES_LIBS_DIR)/java.base/server
 	@$(CP) -p $(OUTPUTDIR)/vm/redirector/$(LIBRARY_PREFIX)jvm_jdk11$(SHARED_LIBRARY_SUFFIX) $(MODULES_LIBS_DIR)/java.base/server/$(LIBRARY_PREFIX)jvm$(SHARED_LIBRARY_SUFFIX)
 
+ifeq (windows,$(OPENJDK_TARGET_OS))
+	@$(ECHO) "Updating vm/lib/jvm.* with vm/j9vm_jdk11/jdk11_jvm.*"
+	@$(CP) -p $(OUTPUTDIR)/vm/j9vm_jdk11/jdk11_jvm$(STATIC_LIBRARY_SUFFIX) $(OUTPUTDIR)/vm/lib/jvm$(STATIC_LIBRARY_SUFFIX)
+	@$(CP) -p $(OUTPUTDIR)/vm/j9vm_jdk11/jdk11_jvm.exp $(OUTPUTDIR)/vm/lib/jvm.exp
```
Comment (keithc-ca) [inline closed/OpenJ9.gmk]: These commands ought to use `$(LIBRARY_PREFIX)` even though it's normally empty.

Comment (pshipton) [inline closed/OpenJ9.gmk]: Does it still work without copying the .exp file?

Comment (ChengJin01) [reply closed/OpenJ9.gmk]: Agreed and added.

File: closed/src/java.base/conf/jvm.cfg [hunk 3]
```diff
@@ -17,6 +17,8 @@
 # 
 # ===========================================================================
 -j9vm KNOWN
+-server IGNORE
```
[the remark this answers is not in the document]
Comment (ChengJin01) [reply closed/src/java.base/conf/jvm.cfg]: We still have the following setting in OpenJ9.gmk to deal with the /server directory 
e.g.
```
	@$(MKDIR) -p $(MODULES_LIBS_DIR)/java.base/server
	@$(CP) -p $(OUTPUTDIR)/vm/redirector/$(LIBRARY_PREFIX)jvm_jdk11$(SHARED_LIBRARY_SUFFIX) $(MODULES_LIBS_DIR)/java.base/server/$(LIBRARY_PREFIX)jvm$(SHARED_LIBRARY_SUFFIX)
```
If the /server should be totally removed, then all setting in jvm.cfg and Copy-java.base.gmk  won't be necessary.

**Comment by pshipton** [comment]
@jdekonin do we have any PR builds for jdk11 yet?

jenkins compile xlinux

File: closed/src/java.base/conf/jvm.cfg (hunk 3 above)
[the remark this answers is not in the document]
Comment (pshipton) [reply closed/src/java.base/conf/jvm.cfg]: Not sure why its different from jdk10, which doesn't IGNORE server.

[the remark this answers is not in the document]
Comment (pshipton) [reply closed/src/java.base/conf/jvm.cfg]: Although the file is there in the jdk10 extensions, perhaps its not used?

[the remark this answers is not in the document]
Comment (pshipton) [reply closed/src/java.base/conf/jvm.cfg]: We create a server directory intentionally, for compatibility with OpenJDK. So an app which looks in the server directory for the libjvm.so will find it and work. I don't want to remove that part, if that is what you meant.

**Comment by pshipton** [comment]
@jdekonin Adam mentioned that only you can run jdk11 builds, however all committers should be able to start them from extensions, or from OpenJ9. They just shouldn't be enabled by default, but must be explicitly requested.

**Comment by pshipton** [comment]
Anyway, I want to ensure these changes for Windows don't break the xLinux build.

File: closed/src/java.base/conf/jvm.cfg (hunk 3 above)
[the remark this answers is not in the document]
Comment (ChengJin01) [reply closed/src/java.base/conf/jvm.cfg]: Yes. So all changes related to jvm.cfg should be kept there in favor of the operations on the server directory.

**Comment by pshipton** [comment]
@ChengJin01 note that once we finalize these changes, they should also be delivered to the "head" stream https://github.com/ibmruntimes/openj9-openjdk-jdk

**Comment by ChengJin01** [comment]
> once we finalize these changes, they should also be delivered to the "head" stream https://github.com/ibmruntimes/openj9-openjdk-jdk

If so,  I need to create a pull request on ibmruntimes/openj9-openjdk-jdk to verify the compilation result.

File: closed/OpenJ9.gmk (hunk 2 above)
Comment (ChengJin01) [reply closed/OpenJ9.gmk]: Already recompiled and it seems there is no issue detected during compilation (also no problem with jdk/bin/java -version). So removed the setting for jvm.exp.

**Comment by AdamBrousseau** [comment]
jenkins compile xlinux

File: closed/OpenJ9.gmk (hunk 2 above)
Comment (keithc-ca) [reply closed/OpenJ9.gmk]: It should have been added to both arguments of `$(CP)`.
